.travis.yml

```diff
@@ -67,14 +67,6 @@ jobs:
       - brew update && brew unlink python@2 && brew install gpgme
       script:
       - hack/travis_osx.sh
-    - stage: local-build
-      <<: *local-build
-      name: local build for linux
-      os: linux
-      services:
-      - docker
-      script: 
-      - make vendor && ./hack/tree_status.sh && make local-cross && make check
```

Comment:
Was `local-cross` (e.g. testing that this can be built even on Windows) dropped intentionally?

Reply:
Hmmm...this might be an "oops".  Should it be added to the `cross` task (running on a mac), `validate` or a new task (so it can run in parallel)?  I'm not sure how long it takes and/or if it should block anything on failure. Please advise.

Reply:
It’s a set of 5 compiles, probably not _too_ expensive; with `go build -a` to remove some of the caching effects, on my local system, 91s of wall clock time, 362s of CPU time, but I didn’t measure this in Cirrus.

I think this should overall block `success_task`, but probably nothing else — if a Windows build fails, that’s good to know but not directly a reason not to run the Linux integration tests; both might fail for different reasons. _Typically_ the most interesting failures happen either early in `validate` (missing sign-offs, bad formatting) or in integration tests, so it would be nice for users to see such failures reasonably fast.

Conceptually it’s a part of the “non-Linux build” checks, i.e. probably `cross_task`. OTOH it is architecture-independent, and makes sense to run this on our cheapest machines, and perhaps to avoid the overhead of extra environment setup / pulls / go module fetches and all that, so adding it to either `validate_task` or `test_skopeo_task` to maximize parallelism and minimize total cost might be worthwhile.

TMTOWTDI:
- Rename `validate_task` to `cheaper_tests_task`, with a `validate_script` and `cross_script`.
- Repurpose `cross_task` to run `local-cross`, and move the existing macOS tests into a `mac_task`.

Many other things would work. Which one do you prefer?

Reply:
> makes sense to run this on our cheapest machines,

Oh! Okay, well that would actually be the `cross_task` - a Mac VM that we pay $0 for :grin: 

> maximize parallelism and minimize total cost might be worthwhile.

Ease of debugging is important too.  This (and c/image) traffic is so low, I'm not afraid to throw another VM at a 5-minute test.  That will make it parallel and easy for anybody to debug.

Opened PR: https://github.com/containers/skopeo/pull/1341